src/cascadia/TerminalSettingsModel/CascadiaSettingsSerialization.cpp

```diff
@@ -1110,8 +1110,9 @@ void CascadiaSettings::WriteSettingsToDisk()
 
     // write current settings to current settings file
     Json::StreamWriterBuilder wbuilder;
-    wbuilder.settings_["indentation"] = "    ";
     wbuilder.settings_["enableYAMLCompatibility"] = true; // suppress spaces around colons
+    wbuilder.settings_["indentation"] = "    ";

```

Comment:
i see you sorted this list ;P

Reply:
If it's more than 1 line it must be sorted alphabetically! No exceptions! 😄